build.xml

```diff
@@ -93,4 +93,12 @@
         </exec>
     </target>
 
+    <!-- This is known to fail but gives an idea of what the hot issues are. -->
+    <target name="run.test.for.testDefinitelyTyped">
```

Comment:
1. In the main menu: Run > Run... (Debug...)
you will get the list of available run configurations
2. Choice what you need and run

You can find more information [here](https://www.jetbrains.com/help/idea/2016.1/run-debug-configuration.html)